.github/workflows/build_client.yml

```diff
@@ -20,10 +20,9 @@ jobs:
       - name: Install npm dependencies
         run: |
           npm install
-      - name: Updating serverBaseURL.
-        run: |
-          find . -type f -name "*build.gradle" -print0 | xargs -0 sed -i "s/api-internal.sandbox.xyz.net/${{ github.event.inputs.defaultServerBaseURL }}/g"
-          find . -type f -name "*gradle.properties" -print0 | xargs -0 sed -i "s/sqp_19c9702eb9f5ddfebbfc36eaa6edf39d22280a49/${{ secrets.SONAR_TOKEN }}/g"
+      - name: Updating libs-snapshot-local to libs-release local for artifactory URL's.
```

Comment:
Why is the sonar token replacement command removed?

Reply:
I was not aware, it was a conflict emerged upon rebase. I will put it back.

Reply:
can you revert this file as well

Reply:
![image](https://github.com/mosip/android-registration-client/assets/56625754/c8dd43cf-9837-43b9-9267-30b3df0cdb3a)